docs/lib/datastore/fragments/android/relational/save-many-snippet.md

```diff
@@ -68,6 +68,33 @@ Amplify.DataStore.save(post,
 )
 ```
 
+</amplify-block>
+<amplify-block name="RxJava">
+
+```java
+Post post = Post.builder()
+    .title("My First Post")
+    .build();
+
+User editor = User.builder()
+    .username("Nadia")
+    .build();
+
+PostEditor postEditor = PostEditor.builder()
+    .post(post)
+    .editor(editor)
+    .build();
+
+Completable.merge(
```

Comment:
:eyes: here please as I made this up.

Reply:
Since `postEditor` depends on `post` and `editor`, I think the backend will fail when saving postEditor, since all 3 of these requests will start at the same time.  This should probably be something like:
```
Completable.merge(
    RxAmplify.DataStore.save(post),
    RxAmplify.DataStore.save(editor)
).andThen(
    RxAmplify.DataStore.save(postEditor)
).subscribe(
    () -> Log.i("MyAmplifyApp", "Post, Editor, and PostEditor saved."),
    failure -> Log.e("MyAmplifyApp", "Item not saved.", failure)
);

Reply:
I think that's correct! I'll test it. Thanks.